openapi-v3.yaml

```diff
@@ -841,7 +868,8 @@ components:
           type: integer
           example: 5000
         dbt_project_subdirectory:
-          type: [null, integer]
+          type: string
+          nullable: true
```

Comment:
Linting & type fix, `nullable: true` is the correct way to specify that a property can be null